docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -32,6 +32,15 @@ const sidebarSections = SIDEBAR[langCode].reduce((col, item) => {
         </div>
       </li>
     ))}
+    <li>
+      <div class="nav-group">
+        <h2 class="nav-group-title">Sponsors</h2>
```

Comment:
Can we work on this design a bit more? It feels odd to have the logo be so small.

Also, what about placing at the top of the sidebar? I don't want to overpower, but I also don't want it to feel too hidden, either. We could move to the right sidebar as well, if that's a better compromise.

played around a bit and this feels promising. We could even make the logo grayscale (until you hover) if we're worried about brand color clashes:
 
<img width="494" alt="Screen Shot 2021-09-14 at 2 13 25 PM" src="https://user-images.githubusercontent.com/622227/133334791-05030f70-90aa-4b9f-a3a8-7bdff6c5c472.png">

Reply:
Some inspiration: https://vuejs.org/v2/guide/

Reply:
I like that design 👆🏻. Updated to reference that. Made the logo a little smaller because I was worried about it pushing menu items too far down. Trust me—at the top of the nav everyone will see it no matter the size.